frontend/app/src/components/staking/kraken/KrakenStakingEvents.vue

```diff
@@ -275,10 +275,10 @@ export default defineComponent({
         orderByAttribute: sortBy[0],
         limit: itemsPerPage,
         offset: (page - 1) * itemsPerPage,
-        fromTimestamp: fromTimestamp,
-        toTimestamp: toTimestamp,
-        eventSubtypes: eventSubtypes ? [eventSubtypes] : undefined,
-        asset: asset || undefined
+        fromTimestamp: fromTimestamp as string,
```

Comment:
Is there a specific reason do we have to explicitly cast to string here?

Reply:
the ts throw error before, because get(filter) values are returned as `string | string[]`

Reply:
hmm, I will have to check in the IDE to see why then, thanks for the pointer

Reply:
OK it makes sense, to change that we would probably need to re-evaluate and refactor the filters